### PR TITLE
media-gfx/gimp: 2.10.38 version bump

### PR DESCRIPTION
**Pull request by band-a-prend**

### CODE REVIEW

**Review by ceamac — COMMENTED** [review]
One test fails `FAIL: test-tools`.  It also fails in 2.10.36-r2, so it's not a regression, but please take a look.  I see there is a bug about this: https://bugs.gentoo.org/910444

[test-suite.log.txt](https://github.com/user-attachments/files/15607010/test-suite.log.txt)

It also blocks stabilization.

Also please run qa-vdb:
```
$ qa-vdb gimp
VDB: detected possibly incorrect RDEPEND (media-gfx/gimp-2.10.38)
                               > app-arch/bzip2
                               > app-arch/xz-utils
app-accessibility/at-spi2-core < 
dev-libs/libgudev:=            | dev-libs/libgudev
dev-libs/libxml2               < 
dev-libs/libxslt               < 
                               > x11-libs/libX11
                               > x11-libs/libXext
                               > x11-libs/libXfixes
                               > x11-libs/libXmu
```
The dependencies on the right are needed in RDEPEND, those on the left may or may not be needed (you need to check).

Some dependencies may also depend on USE flags.

Thank you!

**Comment by band-a-prend** [comment]
@ceamac  
Thanks for review! I had some troubles with fetching from github since last one in early May but now I updated my `dev` Gentoo tree and will try to fix qa-vdb warnings - I didn't used it yet. 

As for  mentioned issue ( https://bugs.gentoo.org/910444 ) I will recheck it again - as I remember it non-regular to reproduce and is observed from time to time on new rebuild/test.

**Comment by band-a-prend** [comment]
@ceamac 
I rechecked that some gimp libraries depends on `app-accessibility/at-spi2-core` at runtime (and on build):
```
ldd /usr/lib64/libgimpui-2.0.so | grep atk
        libatk-1.0.so.0 => /usr/lib64/libatk-1.0.so.0 (0x00007f9d033a3000
ldd /usr/lib64/libgimpwidgets-2.0.so | grep atk
        libatk-1.0.so.0 => /usr/lib64/libatk-1.0.so.0 (0x00007fa45e77d000)
```
also as libraries under `/usr/lib64/gimp/2.0/modules/` directory.

The `app-arch/bzip2` and `app-arch/xz-utils` is required (if are found on build) for compression on the fly at saving files if requested on save.

File: media-gfx/gimp/gimp-2.10.38.ebuild [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+}
+
+src_install() {
+	gnome2_src_install
+
+	# Workaround for bug #321111 to give GIMP the least
+	# precedence on PDF documents by default
+	mv "${ED}"/usr/share/applications/{,zzz-}gimp.desktop || die
+
+	find "${D}" -name '*.la' -type f -delete || die
```
[the remark this answers is not in the document]
Comment (band-a-prend) [reply media-gfx/gimp/gimp-2.10.38.ebuild]: Done

**Review by ceamac — APPROVED** [review]
Good work, thank you!

About the failing test, please try to contact upstream, maybe they have seen it before, because it is blocking stabilization: [bug #917468](https://bugs.gentoo.org/917468) and [bug #910444](https://bugs.gentoo.org/910444).

Dependencies look a lot better, I'll merge this.

**Pull request closed without merging**